docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaSubmissionQueueHandler.php

```diff
@@ -78,35 +84,71 @@ protected function createFoiaRequest(WebformSubmissionInterface $webformSubmissi
       $foiaRequest->set('field_requester_email', $requesterEmailAddress);
     }
 
+    if ($this->fileAttachmentSubmitted($webformSubmission)) {
+      $foiaRequest->setRequestStatus(FoiaRequestInterface::STATUS_SCAN);
+    }
+
     $foiaRequest->save();
     return $foiaRequest;
   }
 
   /**
-   * Adds the FOIA request to the foia_submissions queue.
+   * Determines whether or not any file attachments were submitted.
+   *
+   * @param \Drupal\webform\WebformSubmissionInterface $webformSubmission
+   *   The webform submission.
    *
-   * @param \Drupal\foia_request\Entity\FoiaRequestInterface $foiaRequest
-   *   The FOIA Request to queue for later processing.
+   * @return bool
+   *   TRUE if at least one attachment was submitted, otherwise FALSE.
    */
-  protected function queueFoiaRequest(FoiaRequestInterface $foiaRequest) {
-    /** @var \Drupal\Core\Queue\QueueFactory $queueFactory */
-    $queueFactory = \Drupal::service('queue');
-
-    // @var QueueInterface $queue
-    $foiaSubmissionsQueue = $queueFactory->get('foia_submissions');
-    $submission = new \stdClass();
-    $submission->id = $foiaRequest->id();
+  protected function fileAttachmentSubmitted(WebformSubmissionInterface $webformSubmission) {
+    $webform = $webformSubmission->getWebform();
+    if ($webform->hasManagedFile()) {
+      $fileAttachmentElementsOnWebform = $this->getFileAttachmentElementsOnWebform($webform);
+      return $this->fileAttachmentsExist($fileAttachmentElementsOnWebform, $webformSubmission);
```

Comment:
Should this return TRUE like on line 148?

Reply:
`$this->fileAttachmentsExists()` returns TRUE or FALSE.